userguide/intro.rst

```diff
@@ -223,6 +223,10 @@ These screen options have changed:
 * The :guilabel:`Periodic Notification User` field has been removed in
   :menuselection:`System --> Advanced`.
 
+* The new
+  :menuselection:`System --> Reporting`
```

Comment:
Use a `:ref:` here instead of menuselection to give the user a link to the new section.